src/controllers/subscribeController.js

```diff
@@ -0,0 +1,18 @@
+const subscribeService = require("../services/subscribeService");
+const { subscriptionCheck, showSubscription } = subscribeService;
+
+const showSubscriptionPlans = async (req, res) => {
```

Comment:
```suggestion
const getSubscriptionPlans = async (req, res) => {
```
'show' 보다는 'get'이 보다 일반적인 이름입니다 :)

Reply:
함수명 변경하였습니다!
+) 모듈을 잘못 불러오는 구간이 있던 탓에 테스트 코드에 오류가 있어서 수정해두었습니다!